### PR TITLE
Small Fixes Bundle

### PR DESCRIPTION
**Pull request by manuelsc**

### CODE REVIEW

**Comment by D13ce** [comment]
`mipmap-anydpi-v26` is empty, can it be removed or does it require the same resources as, for example, `mipmap-hdpi`?

File: src/app/controllers/MachineController.ts [hunk 1]
```diff
@@ -205,67 +205,12 @@ export default class MachineController {
 		} as MachineChartData
 	}
 
-	public getFallbackConfigurations(data: ProcessedStats): FallbackConfigurations {
```
Comment (D13ce) [inline src/app/controllers/MachineController.ts:208]: Removing the entire fallback handling made both `sync_eth2_fallback_configured` and `sync_eth2_fallback_connected` in `interface ProcessBase` obsolete. They should be removed.

File: src/app/pages/machines/machines.page.html [hunk 2]
```diff
@@ -94,7 +94,7 @@
 			<ion-label><strong class="title">Machine Monitoring</strong></ion-label
 			><br /><br />
 
-			<ion-label>Monitor your eth2 staking machines on the go.</ion-label>
+			<ion-label>Monitor your ethereum staking machines on the go.</ion-label>
```
Comment (D13ce) [inline src/app/pages/machines/machines.page.html:97]: "ethereum" should be capitalized.

**Comment by D13ce** [comment]
I have found a bug with the new income chart handling:
image
As you can see, the maximum of the charts y-axis is set to 0.132 yet the selected bar has a value of 0.155 i.e. it exceeds the maximum.

File: src/app/pages/machines/machines.page.html (hunk 2 above)
Comment (manuelsc) [reply src/app/pages/machines/machines.page.html]: done

**Comment by D13ce** [comment]
@manuelsc, as discussed, I have used this branch to fix yet another chart related bug (BIDS-1208).
Please review commit e78417bf4716af53a65d33a9e6d229f45c73d40c and close this conversation if everything's looking good.

**Comment by manuelsc** [comment]
lgtm

**Review by D13ce — APPROVED** [review]
lgtm

**Pull request merged**